src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -3,7 +3,7 @@
 namespace Illuminate\Foundation\Exceptions;
 
 use Exception;
-use Psr\Log\LoggerInterface;
+use Illuminate\Contracts\Logging\Log;
```

Comment:
we don't need a Illuminate\Contracts\Logging\Log, any Psr\Log\LoggerInterface is sufficient